friends/src/components/Friend.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Friend = props => {
+  const { name, age, email } = props.friend;
+
+  return(
+    <div className="friend-card">
+
+      <div>
```

Comment:
Using `<p>` tags might be more appropriate than `<div>` tags for the name, age, and email elements.